src/org/labkey/targetedms/outliers/OutlierGenerator.java

```diff
@@ -150,28 +150,34 @@ private String getEachSeriesTypePlotDataSql(int seriesIndex, QCMetricConfigurati
         return sql.toString();
     }
 
-    /** @return LabKey SQL to fetch all the values for the specified metrics */
-    private String queryContainerSampleFileRawData(List<QCMetricConfiguration> configurations, Date startDate,
-                                                   Date endDate, List<AnnotationGroup> annotationGroups,
-                                                   boolean showExcluded)
+    private Map<String, QCMetricConfiguration> getPreferredMetrics(List<QCMetricConfiguration> configurations, boolean forOutliers)
```

Comment:
I suggest renaming `forOutliers` to `forOutlierSummary` since both are used for outliers, just in different ways.